packages/ai/src/openai/azure.ts

```diff
@@ -589,6 +593,89 @@ export class WrappedResponses extends AzureOpenAI.Responses {
   }
 }
 
+export class WrappedEmbeddings extends AzureOpenAI.Embeddings {
+  private readonly phClient: PostHog
+
+  constructor(client: AzureOpenAI, phClient: PostHog) {
+    super(client)
+    this.phClient = phClient
+  }
+
+  public create(
+    body: EmbeddingCreateParams & MonitoringParams,
+    options?: RequestOptions
+  ): APIPromise<CreateEmbeddingResponse> {
+    const {
+      posthogDistinctId,
+      posthogTraceId,
+      posthogProperties,
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do we need this disable? If we remove it, can we easily fix what it's complaining about?